src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -429,21 +429,24 @@ protected async Task<LaunchSettingsData> ReadSettingsFileFromDiskAsync()
             // serialize their section. Unfortunately, this means the data is string to object which is messy to deal with
             var launchSettingsData = new LaunchSettingsData() { OtherSettings = new Dictionary<string, object>(StringComparers.LaunchProfileProperties) };
             var jsonObject = JObject.Parse(jsonString);
```

Comment:
Design question to @drewnoakes and @tmeschter: Is there a reason we're doing this custom type conversion instead of making a JSON type converter? I'd recommend using System.Text.Json instead of Newtsonsoft Json for the performance benefits. A type converter for System.Text.Json is documented here: https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-converters-how-to?pivots=dotnet-5-0#steps-to-follow-the-factory-pattern

System.Text.Json is in the non-framework versions of .NET but the package exists that works for 4.7.2 and up: https://www.nuget.org/packages/System.Text.Json
Docs: https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to?pivots=dotnet-5-0